src/main/steps/applicant1/hub-page/joint/content.ts

```diff
@@ -182,12 +186,13 @@ const cy: typeof en = ({ isDivorce, userCase, partner, isApplicant2 }: CommonCon
     )} ar ôl i'ch cais gael ei wirio.
     Bydd yn cynnwys yr amser, y dyddiad a manylion y llys lle bydd eich gorchymyn amodol yn cael ei gyhoeddi.`,
   },
-  awaitingFinalOrder: {
-    line1: `Gallwch nawr wneud cais am 'orchymyn terfynol'. Gorchymyn terfynol yw'r ddogfen a fydd yn dod â'ch ${
-      isDivorce ? 'priodas' : 'partneriaeth sifil'
-    } i ben yn gyfreithiol. Dyma'r cam olaf yn y ${
-      isDivorce ? 'broses ysgaru' : "broses i ddod â'ch partneriaeth sifil i ben"
-    }.`,
+  awaitingFinalOrderOrFinalOrderOverdue: {
+    line1: `You can now apply for a 'final order'. A final order is the document that will legally end your ${
```

Comment:
why undo the welsh? it doesnt seem like the text in English has changed.

Reply:
I reverted that, but they don't have a translation for the button text.

Reply:
Thats fine, I assume they'll be a ticket for this. might be worth double checking.